Assets/AppCenter/Editor/AppCenterPreBuild.cs

```diff
@@ -49,6 +55,48 @@ public void OnPreprocessBuild(BuildTarget target, string path)
         }
     }
 
+#if UNITY_ANDROID
+    public static void AddSettingsFileToLoader(AppCenterSettingsMakerAndroid settingsMaker)
+    {
+        var loaderZipFile = AppCenterSettingsContext.AppCenterPath + "/Plugins/Android/appcenter-loader-release.aar";
+        var loaderFolder = "appcenter-loader-release";
+        var settingsFilePath = "appcenter-loader-release/res/values/appcenter-settings.xml";
+        var settingsMetaFilePath = "appcenter-loader-release/res/values/appcenter-settings.xml.meta";
+
+        if (!File.Exists(loaderZipFile))
+        {
+            Debug.LogWarning("Failed to load dependency file appcenter-loader-release.aar");
+            return;
+        }
+
+        // Delete unzipeed directory if it already exists.
+        if (Directory.Exists(loaderFolder))
+        {
+            Directory.Delete(loaderFolder, true);
+        }
+
+        AndroidLibraryHelper.UnzipFile(loaderZipFile, loaderFolder);
+        if (!Directory.Exists(loaderFolder))
+        {
+            Debug.LogWarning("Unzipping loader folder failed.");
+            return;
+        }
+
+        settingsMaker.CommitSettings(settingsFilePath);
+
+        // Delete the appcenter-settings.xml.meta file if generated
```

Comment:
End comments with dots.
```suggestion
        // Delete the appcenter-settings.xml.meta file if generated.
```